### ISSUE TITLE
User information encoding seems to be lost

### ISSUE THREAD

**Issue by peterblazejewicz** [auth0/auth0-spa-js#27]
Not sure if that is a correct place. It seems that at some point the encoding of the user information is being lost. Not sure if that is  library specific, but I've noticed that on my profile page while using auth0-samples/auth0-spa-js-angular-samples locally.

Here is GitHub SDK call:

```bash
curl https://api.github.com/users/peterblazejewicz
```
returning information:

```json
"name": "Piotr Błażejewicz (Peter Blazejewicz)",
"company": ...
```

while the library keeps information as the:

```json
{
  "name": "Piotr BÅaÅ¼ejewicz (Peter Blazejewicz)",
  "updated_at": ...
}
```

Thanks!

**Comment by luisrudge** [comment]
Maybe it's the sample encoding tag? All the SDK does is relay the server information to you. If you inspect the requests int the dev tools, do you get the correct information?

**Comment by peterblazejewicz** [comment]
When I copy the returned id token to the debugger here: https://jwt.io, I'm seeing correctly decoded strings:

```json
{
  "nickname": "peterblazejewicz",
  "name": "Piotr Błażejewicz (Peter Blazejewicz)",
  "picture": "https://avatars3.githubusercontent.com/u/14539?v=4",
  "updated_at": "2019-06-15T19:57:56.447Z",
  "email": "{{}}",
  "email_verified": true,
  "iss": "https://blazejewicz.eu.auth0.com/",
  "sub": "github|14539",
  "aud": "Yo9lLpD5iyVxJApE12HQX2FrN8W4PWZr",
  "iat": 1560628678,
  "exp": 1560664678,
  "nonce": "YqkOvc8n1XqH~NDBOOTydGGscuoLnvo1hTRC8Q.Cji1"
}
```
while auth0 client `getUser` returns:

```json
email: "{{}}"
email_verified: true
name: "Piotr BÅaÅ¼ejewicz (Peter Blazejewicz)"
nickname: "peterblazejewicz"
picture: "https://avatars3.githubusercontent.com/u/14539?v=4"
sub: "github|14539"
updated_at: "2019-06-15T20:13:01.371Z"
```

I assume it's being decoded on the client using the library here. The page bears the UTF-8

**Comment by luisrudge** [comment]
Thanks for the help and context, @peterblazejewicz. I opened a PR to fix this #28

**Comment by peterblazejewicz** [comment]
You're welcome!

**Comment by luisrudge** [comment]
released in @auth0/auth0-spa-js@1.0.0-beta.3 🎉
